lib/ansible/modules/command.py

```diff
@@ -86,9 +91,9 @@
     -  If you want to run a command through the shell (say you are using C(<), C(>), C(|), etc), you actually want the M(ansible.builtin.shell) module instead.
        Parsing shell metacharacters can lead to unexpected commands being executed if quoting is not done correctly so it is more secure to
        use the C(command) module when possible.
-    -  " C(creates), C(removes), and C(chdir) can be specified after the command.
+    -  " C(creates), C(removes), C(changes) and C(chdir) can be specified after the command.
        For instance, if you only want to run a command if a certain file does not exist, use this."
-    -  Check mode is supported when passing C(creates) or C(removes). If running in check mode and either of these are specified, the module will
+    -  Check mode is supported when passing C(creates), C(removes) or C(changes). If running in check mode and either of these are specified, the module will
```

Comment:
It may be worth clarifying here that `changes=/path/to/file` in check mode will always report `changed=True` even if the command would not result in changes during normal execution. There is no way around this.
```suggestion
    -  Check mode is supported when passing C(creates), C(removes) or C(changes). If running in check mode and any of these are specified, the module will
```

Reply:
The alternative would be to always report `changed=False` in check mode, but that could result in an unexpected change during normal execution when you were expecting no change. I would rather Ansible report a change in check mode, but do nothing in normal mode to be on the safe side.

Reply:
We need to double check this as the module always returns `False` as it never knows if the file is going to change. 

Do you think it is better to remove from `check mode`?